starfish/multiprocessing/pool.py

```diff
@@ -0,0 +1,26 @@
+import multiprocessing.pool as mp
+from typing import Optional
+
+
+class Pool(mp.Pool):
+    """Wrapper class for multiprocessing pool. If n_processes=1 just use map on main
+    thread for debugging purposes """
```

Comment:
main process.

it may also be worth documenting a couple of non-trivial differences in behavior between mp.Pool and this for n_processes==1:

1. Depending on the forking model, global state may not exist in mp.Pool whereas they do exist if we just map on main process..
2. initializer gets called on every invocation of mp.Pool.{imap,map}.